Packaging/RedHat/0.build_libraries.sh

```diff
@@ -60,11 +60,11 @@ if [ $? -eq 1 ]; then
 fi
 
 # GNUstep
-`dirname $0`/build_nextspace-gnustep.sh $1
-if [ $? -eq 1 ]; then
-    echo "Aborting..."
-    exit 1
-fi
+#`dirname $0`/1.build_nextspace-gnustep.sh $1
+#if [ $? -eq 1 ]; then
+#    echo "Aborting..."
+#    exit 1
+#fi
 
```

Comment:
Why have you commented this? 1.build_nextspace-gnustep.sh builds -gui and -back from specific nextspace branch.

Reply:
I commented on the lines because in a sequence 0,1,2,3, within the 0 there was the call to 1, then in an automation the call to 1 is repeated.

Reply:
Got it. I need to rename nextspace-gnustep build script back to old name.